hack/update-staging-godeps-dockerized.sh

```diff
@@ -87,6 +87,15 @@ function diffGodepManifest() {
   fi
 }
 
+# Create a fake git repo for staging to prevent godeps from complaining
+pushd "${KUBE_ROOT}/staging/src" >/dev/null
```

Comment:
@dims So I was thinking about this solution some more, particularly in the context of changing the `hack/verify-godeps.sh` script too.. we'd need to do this in root as well, so perhaps it's worth being consistent and just run this at `"${KUBE_ROOT}"`? Or would you mind if this was changed to `"${KUBE_ROOT}"` at a later date?